streamer/configuration.py

```diff
@@ -561,5 +561,5 @@ def validate(cls, key):
 
     if key not in cls.map_class.keys():
       raise ValueError(
-          '{} is not a valid {}'.format(key, cls.map_class.__name__))
+          '{} is not a valid {}, {}'.format(key, cls.map_class.__name__, cls.name()))
```

Comment:
From what I can tell re-reading the source, this now repeats cls.map_class.__name__, which is also printed by name() above:

"foo is not a valid VideoResolution, VideoResolution name (one of 720p, 1080p, ...)"

Should we drop cls.map_class.__name__ in favor of cls.name()?

Reply:
You read that write.
Yes i think we should drop it in favor of `cls.name()`.